src/commands/games/leagueoflegends/skin.js

```diff
@@ -0,0 +1,32 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+
+module.exports = class LeagueOfLegendsSkin extends Command {
+  constructor (client) {
+    super({
+      name: 'skin',
+      aliases: ['s'],
+      parent: 'leagueoflegends',
+      parameters: [{
+        type: 'string', full: true, missingError: 'commands:leagueoflegends.subcommands.skin.noSkin'
+      }]
+    }, client)
+  }
+
+  async run ({ t, author, channel, language }, skin) {
+    channel.startTyping()
+    const embed = new SwitchbladeEmbed(author)
+    try {
+      const split = skin.split('|')
```

Comment:
what is this even???

that's ugly af

Reply:
how do you think that should be made?
1. there are skins that change the name of the original champ (e.g. Beemo)
2. skins that come after the champions name aren't going to work (e.g. True Damage Senna Prestige Edition)

Reply:
i do have an idea, and i'll work that 2nite if i can get the install working

Reply:
Use fuze!

Reply:
ok thats a better idea

Reply:
there was no need in using fuse, but it now works!